inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -116,7 +116,16 @@ def is_record_relevant(obj, *args, **kwargs):
 
 
 def is_experimental_paper(obj, eng):
-    """Is the record an experimental paper."""
-    categories = get_value(obj.data, "arxiv_eprints.categories", []) + \
+    """Check if the record is an experimental paper."""
+    categories = get_value(obj.data, "arxiv_eprints.categories", [[]])[0] + \
         get_value(obj.data, "subject_terms.term", [])
-    return "hep-ex" in categories or "Experiment-HEP" in categories
+    categories_to_check = [
+        "hep-ex", "nucl-ex", "astro-ph", "astro-ph.IM", "astro-ph.CO",
+        "astro-ph.EP", "astro-ph.GA", "astro-ph.HE", "astro-ph.SR",
+        "physics.ins-det",
+        "Experiment-HEP", "Experiment-Nucl", "Astrophysics", "Instrumentation"
+    ]
+    for experimental_category in categories_to_check:
+        if experimental_category in categories:
+            return True
+    return False
```

Comment:
Just one question: Is there a minimal threshold under which the experiment guessed is discarded anyway?